flink-ml-lib/src/main/java/org/apache/flink/ml/feature/variancethresholdselector/VarianceThresholdSelector.java

```diff
@@ -63,11 +63,11 @@ public VarianceThresholdSelectorModel fit(Table... inputs) {
         final String inputCol = getInputCol();
         StreamTableEnvironment tEnv =
                 (StreamTableEnvironment) ((TableImpl) inputs[0]).getTableEnvironment();
-        DataStream<DenseVector> inputData =
+        DataStream<Vector> inputData =
```

Comment:
Let's describe the changes made to `VarianceThresholdSelector` in this PR's description.